workbench/runtime.py

```diff
@@ -164,6 +162,7 @@ def get_block_type(self, def_id):
             raise NoSuchDefinition(repr(def_id))
 
     def create_aside(self, definition_id, usage_id, aside_type):
+        """Create asides"""
```

Comment:
The changes in this class were all to get quality tests passing - not sure why they weren't fixed before, but I figured it can't hurt to fix them now.